src/client/components/Headline/Headline.tsx

```diff
@@ -11,9 +12,13 @@ export type Props = {
   children: React.ReactNode
 }
 
-const HeadlineXL = styled.h2`
+const baseStyles = css`
```

Comment:
Great! 🎉 Isn't the `font-weight` the same for all as well? :)

Reply:
Yep, but that's something that feels like there's a high chance we want to change? But maybe I'm thinking about it wrong haha 🙈

Reply:
I don't think we use any bold text right, or at least we shouldn't looking at the brand guidelines :D
<img width="496" alt="Screenshot 2022-01-13 at 16 44 57" src="https://user-images.githubusercontent.com/6661511/149362132-b719eb66-dbac-49eb-8d5a-4f72b334ac01.png">

Reply:
Ah OK, cool! 👍  Where are these guidelines? Haven't seen them 😳